mavros/src/plugins/setpoint_trajectory.cpp

```diff
@@ -183,7 +183,8 @@ class SetpointTrajectoryPlugin : public plugin::Plugin,
       return;
     }
 
-    Eigen::Vector3d position, velocity, af;
+    Eigen::Vector3d position = Eigen::Vector3d::Zero(), velocity = Eigen::Vector3d::Zero(),
+      af = Eigen::Vector3d::Zero();
```

Comment:
AFAIK default constructor make zeroes anyway.

Reply:
The compiler was giving warnings.

Reply:
Ok.